### PR TITLE
Feature/osint 273 (maigret)

### PR DESCRIPTION
**Pull request by ahrufox**
# [OSINT-273] maigret
## 🔑 Key Change 
- Maigret result with label, property, type, value

## 🖼️ Screenshots (if necessary)
image

## 🙏 To Reviewers
🧑‍🤝‍🧑 @Teammate
- remark 1

[OSINT-273]: https://bob-transfer.atlassian.net/browse/OSINT-273?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by D7MeKz — APPROVED** [review]
👍

File: server/osint/tools/lib/tool_maigret.py [hunk 1]
```diff
@@ -46,9 +46,15 @@ def report_maigret(case_id, run):  # status is COMPLETED
...
     for i in range(length):
+        # inside = {
+        #     "site": result_site[i],
+        #     "url_user": result_url[i]
+        # }
         inside = {
-            "site": result_site[i],
-            "url_user": result_url[i]
+            "label": "SurfaceUser",
+            "property": "others",
```
Comment (R3D1NM) [inline server/osint/tools/lib/tool_maigret.py:55]: registered임

**Review by R3D1NM — CHANGES_REQUESTED** [review]
SurfaceUser에 registered로 들어갑니다

File: server/osint/tools/lib/tool_maigret.py (hunk 1 above)
Comment (ahrufox) [reply server/osint/tools/lib/tool_maigret.py]: fixed

**Review by R3D1NM — APPROVED** [review]
👍

**Pull request merged**